app/gateway/2.7.x/admin-api/consumer-groups/reference.md

```diff
@@ -0,0 +1,333 @@
+
+**Endpoint**
+<div class="endpoint delete">/consumer-groups/{GROUP_NAME|GROUP_ID}</div>
+
+Attribute                             | Description
+---------:                            | --------
+`GROUP_NAME|GROUP_ID`<br>*required*   | The name or UUID of the consumer group to delete.
+
+## Remove a consumer
+
+### Remove a consumer from all groups
+
+**Endpoint**
+<div class="endpoint delete">/consumers/{USERNAME|CONSUMER_ID}/consumer_groups</div>
+
+Attribute                             | Description
+---------:                            | --------
+`USERNAME|CONSUMER_ID`<br>*required*   | The name or UUID of the consumer to remove from all groups.
+
+**Response**
+
+```
+HTTP/1.1 204 No Content
+```
+
+### Remove a consumer from a specific group
+
+**Consumer endpoint**
+<div class="endpoint delete">/consumers/{USERNAME|CONSUMER_ID}/consumer_groups/{GROUP_NAME|GROUP_ID}</div>
+
+Attribute                             | Description
+---------:                            | --------  
+`USERNAME|CONSUMER_ID`<br>*required*  | The name or UUID of the consumer to remove.
+`GROUP_NAME|GROUP_ID`<br>*required*   | The name or UUID of the consumer group to remove the consumer from.
+
+**Response**
+
+```
+HTTP/1.1 204 No Content
+```
+
+**Consumer groups endpoint**
+<div class="endpoint delete">/consumer_groups/{GROUP_NAME|GROUP_ID}/consumers/{USERNAME|CONSUMER_ID}</div>
+
+Attribute                             | Description
+---------:                            | --------    
+`GROUP_NAME|GROUP_ID`<br>*required*   | The name or UUID of the consumer group to remove the consumer from.
+`USERNAME|CONSUMER_ID`<br>*required*  | The name or UUID of the consumer to remove.
+
+**Response**
+
+```
+HTTP/1.1 204 No Content
+```
+
+## Configure rate limiting for a consumer group
+
+<div class="endpoint put">/consumer_groups/{GROUP_NAME|GROUP_ID}/overrides/plugins/rate-limiting-advanced</div>
+
+Attribute                             | Description
+---------:                            | --------    
+`GROUP_NAME|GROUP_ID`<br>*required*   | The name or UUID of the consumer group to configure.
+
+**Request Body**
+
+Attribute     | Description  
+---------:    | --------
+`limit`<br>*required*       | An array of one or more requests-per-window limits to apply. There must be a matching number of window limits and sizes specified.
+`window_size`<br>*required* | An array of one or more window sizes to apply a limit to (defined in seconds). There must be a matching number of window limits and sizes specified.
+`window_type`<br>*optional* | Set the time window type to either `sliding` (default) or `fixed`.
+`retry_after_jitter_max`<br>*optional* | The upper bound of a jitter (random delay) in seconds to be added to the `Retry-After` header of denied requests (status = `429`) in order to prevent all the clients from coming back at the same time. The lower bound of the jitter is `0`; in this case, the `Retry-After` header is equal to the `RateLimit-Reset` header.
```

Comment:
Are these all of the possible editable params for this endpoint?

Reply:
Yes.